satpy/etc/readers/fci_l2_nc.yaml

```diff
@@ -2196,10 +2425,11 @@ datasets:
     wavelength: [0.894, 0.914, 0.934]
     file_type: nc_fci_asr
     file_key: quality_reflectance
-    long_name: quality_reflectance_all
     vis_channel_id: 4
     category_id: 0
     fill_value: -1
+    long_name: TOA Bidirectional Reflectance % confidence at 0.9um (all pixels)
```

Comment:
I think it would be good to have two decimals for the wavelength here, for three reasons:
1. Better precision.
2. the 0.865 and 0.914 channel are now both named 0.9.
3. The 1.38 channel is usually referred to as the 1.3 channel. Here, it's called 1.4um however.

Reply:
Done